packages/sql-openapi/test/simple.test.js

```diff
@@ -466,3 +464,43 @@ test('simple db, simple rest API', async (t) => {
   const json = res.json()
   matchSnapshot(json, 'GET /documentation/json response')
 })
+
+test('deserialize JSON columns', { skip: isSQLite }, async (t) => {
```

Comment:
Are you sure MariaDB, MySQL, and MySQL 5.7 supports this?

Reply:
MariaDB converts JSON to `longtext` so in Platformatic would be a `string` . Anyway I added tests specifically for MariaDB